src/controllers/TalksController.php

```diff
@@ -289,6 +367,12 @@ public function createTalkAction(Request $request, PDO $db)
         $tz = new DateTimeZone($event['tz_continent'] . '/' . $event['tz_place']);
         $talk['date'] = (new DateTime($start_date, $tz))->format('U');
 
+        $event_start_date = (new DateTime($event['start_date']))->format('U');
```

Comment:
This is just moving existing code to a function for use by both add and edit talk. Original code is here: https://github.com/joindin/joindin-api/blob/master/src/controllers/TalksController.php#L390

i.e. separate ticket to determine if this line is correct or not.